rpmconf/Install/zmsetup.pl

```diff
@@ -5787,6 +5789,27 @@ sub configCreateCert {
     }
   }
 
+  if (isInstalled("zimbra-license-daemon")) {
+	  if ( !-f "/opt/zimbra/ssl/zimbra/server/server.crt") {
+		  progress ( "Creating SSL certificate..." );
+		  $rc = runAsZimbra("/opt/zimbra/bin/zmcertmgr createcrt $needNewCert");
```

Comment:
can we move 
`		  $rc = runAsZimbra("/opt/zimbra/bin/zmcertmgr createcrt $needNewCert");
		  if ($rc != 0) {
			  progress ( "failed.\n" );
			  exit 1;
		  } else {
			  progress ( "done.\n" );
		  }
` 
out of if-else block as thats getting repeated

Reply:
there may be a case if it not goes into the IF...ELSIF block